GIF-Swift/iOSDevCenters+GIF.swift

```diff
@@ -62,8 +45,9 @@ extension UIImage {
         return gifImageWithData(imageData)
     }
     
-    class func delayForImageAtIndex(_ index: Int, source: CGImageSource!) -> Double {
-        var delay = 0.1
+    private class func delayForImageAtIndex(_ index: Int, source: CGImageSource!) -> Double {
+        let defaultDelay = 0.032 // 30 fps
```

Comment:
`0.032` value could be even less, maybe 0.001. In my case, per frame delay was 0.02 for a gif.